command/assets/example.nomad

```diff
@@ -327,19 +327,25 @@ job "example" {
       # For more information and examples on the "service" stanza, please see
       # the online documentation at:
       #
-      #     https://www.nomadproject.io/docs/job-specification/service.html
+      #     https://www.nomadproject.io/docs/job-specification/service
       #
       service {
         name = "redis-cache"
         tags = ["global", "cache"]
         port = "db"
 
-        check {
-          name     = "alive"
-          type     = "tcp"
-          interval = "10s"
-          timeout  = "2s"
-        }
+        # The "check" stanza instructs Nomad to create a Consul health check for
+        # this service. A sample check is provided here for your convenience;
+        # uncomment it to enable it. The "check" stanza is documented in the
+        # "service" stanza documentation.
```

Comment:
We're linking all over the place in this doc anyways... should we have a link to that doc page here too?

Reply:
Do you mean linking to `service` or to `check`?  If to service, there's a link pretty close at the top of the service stanza itself, would that be adequate?  For linking to check, it's inside of the service page at a generated ID.  We _could_ link there, but I'm more worried about link rot on #ids.

Reply:
Oh, I missed that `service` link. Let's leave it as-is.